validator/sawtooth_validator/journal/chain.py

```diff
@@ -348,7 +346,7 @@ def run(self):
                 return
 
             # 4) Evaluate the 2 chains to see which is the one true chain.
```

Comment:
This comment doesn't quite make sense anymore, as the "2 chains" are no longer local passed into this function.  In fact, it's no longer clear how two forks are getting into this function. Perhaps a better function name is needed, like `is_forked()` (but maybe better?).